python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -597,6 +612,33 @@ def func(x):
 
         return self.agg(func)
 
+    def describe(self):
```

Comment:
This interface isn't exactly like Pandas right? https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.groupby.DataFrameGroupBy.describe.html

We should be clear what to support and what not to in our interface.

Reply:
> cuDF implementation only works for numeric DataFrameGroupBy whereas Pandas works for Series and DataFrames
I don't know if that's true, your current implementation should work for Series:
```python
>>> gs = gdf['Score']
>>> keys = gdf['Speed']
>>> gs.groupby(keys).describe()
       count  mean   std  min   25%   50%   75%  max
Speed                                               
24.0       1  90.0  <NA>   90  90.0  90.0  90.0   90
26.0       1  80.0  <NA>   80  80.0  80.0  80.0   80
370.0      1  30.0  <NA>   30  30.0  30.0  30.0   30
380.0      1  50.0  <NA>   50  50.0  50.0  50.0   50
```